android/app/build.gradle

```diff
@@ -26,7 +26,24 @@ android {
         targetSdkVersion rootProject.ext.targetSdkVersion
 
         ndk {
-            abiFilters 'armeabi-v7a', 'arm64-v8a', 'x86', 'x86_64'
+            abiFilters 'armeabi-v7a', 'arm64-v8a', 'x86'
```

Comment:
I am not sure, but I recall we discussed about removing support for old arch.

Reply:
Not really, let's keep it.